app/src/components/staking/DelegationsOverview.vue

```diff
@@ -90,18 +88,8 @@ export default {
       const stakingDenomBalance = this.balances.find(
         ({ denom }) => denom === this.currentNetwork.stakingDenom
       )
-      let stakedAmount =
-        Number(stakingDenomBalance.total) -
-        Number(stakingDenomBalance.available)
-      // substract the already unbonding balance in the case of Substrate networks.
-      if (this.undelegationsLoaded && this.undelegations.length > 0) {
-        stakedAmount = this.undelegations.reduce(
-          (stakedAmount, { amount }) => stakedAmount - Number(amount),
-          stakedAmount
-        )
-      }
       return {
-        total: stakedAmount.toFixed(3),
+        total: stakingDenomBalance ? stakingDenomBalance.staked.toFixed(3) : 0,
```

Comment:
I'm a bit confused here. How does this fixes undelegations disappearing after reaching the claimable date?

Reply:
you could read the PR description: Show staked balance based on actual staking ledger and undelegations